src/main/java/com/google/jenkins/plugins/computeengine/InstanceConfiguration.java

```diff
@@ -492,9 +509,36 @@ private Tags tags() {
         return null;
     }
 
-    private Scheduling scheduling() {
+    @VisibleForTesting
+    Scheduling scheduling() {
```

Comment:
We normally handle configuration migration by implementing `readResolve` and making the old fields `transient` and `@Deprecated`. 
See https://www.jenkins.io/doc/developer/persistence/backward-compatibility/

That way the next time the configuration is saved the new format will be on disk.

Reply:
thanks a lot for telling me about it, never knew such feature existed 😲 . makes sense.
fixed in https://github.com/jenkinsci/google-compute-engine-plugin/pull/492/commits/3e7e7872cb2989de02b2201ab92c8e9d27931281